cluster/syncer/statefulset.go

```diff
@@ -258,6 +262,13 @@ func (s *StatefulSetSyncer) updatePod(ctx context.Context) error {
 // 5. Check followerHost current role.
 // 6. If followerHost is not leader, switch it to leader through xenon.
 func (s *StatefulSetSyncer) preUpdate(ctx context.Context, leader, follower string) error {
+	sqlRunner, closeConn, err := s.SQLRunnerFactory(internal.NewConfigFromClusterKey(
```

Comment:
sqlRunner should change name to leaderRunner， indicating that is used for leader